src/Event/EventCreatedEvent.php

```diff
@@ -2,37 +2,6 @@
 
 namespace AppBundle\Event;
 
-use AppBundle\Committee\CommitteeEvent as BaseEvent;
-use AppBundle\Entity\Adherent;
-use AppBundle\Entity\Committee;
-use AppBundle\Entity\Event;
-use AppBundle\Geocoder\GeocodableInterface;
-
-class EventCreatedEvent extends BaseEvent
+class EventCreatedEvent extends EventBaseEvent
```

Comment:
As a general comment for the PR : an "event object" should not be aware of the type of event (=event name).

The goal of an event object is to carry the information about the subject of the dispatched event.

I.e. :

```
$this->dispatcher->dispatch(Events::USER_CREATED, new UserEvent($user));
$this->dispatcher->dispatch(Events::USER_UPDATED, new UserEvent($user));
```

At the end we should have one event object that carries a committee and one event object that carries an event (related to EM stuff).